features/yield/BalancerData.tsx

```diff
@@ -60,7 +62,7 @@ const BalancerData = () => {
     return (
       <span>
         <Typography variant="h5" style={{ marginBottom: "10px" }}>
-          yUSD Pool Metrics{" "}
+          {tokenSymbol} Pool Metrics{" "}
```

Comment:
Does this render something strange if the selected token is a non yield token?

Reply:
Balancer data should only show up for a yield token. This page wont load if the token is not a yield token.